addons/crm/models/crm_team_member.py

```diff
@@ -15,7 +15,7 @@
 _logger = logging.getLogger(__name__)
 
 
-class Team(models.Model):
+class TeamMember(models.Model):
```

Comment:
One small typo in your commit message, "automaticallyK"

Reply:
Good catch !